packages/core-js/utils/ast/finders.js

```diff
@@ -172,6 +182,26 @@ function findFunctionSelectors(contractName, astNodes) {
   return selectors;
 }
 
+/**
+ * Get all the function definitions from the complete tree of contract
+ * nodes starting from the given root contract definition
+ * @param {string} contractName
+ * @param {import("solidity-ast").SourceUnit[]} astNodes
+ * @returns {import("solidity-ast").FunctionDefinition[]}
+ */
+function findFunctions(contractName, astNodes) {
+  const functions = [];
+
+  for (const contractNode of findContractDependencies(contractName, astNodes)) {
+    const currentFunctions = _findFunctions(contractNode);
```

Comment:
IMHO, it would be clearer to delete the `_findFunctions`  fn and replace it for this:
```suggestion
    const currentFunctions = Array.from(findAll('FunctionDefinition', contractNode));
```

Reply:
implemented